sdk/pull-requests.md

```diff
@@ -14,6 +14,34 @@ This document is intended as a canonical reference documenting how the SDK Team
 - Ideally, in most cases, PRs should be landed (merged) to the `main` branch by the Lead Engineer responsible for that client library
 - Avoid publishing comments into that public domain that just represent housekeeping or reminders to other team members - these should more appropriately be handled as internal messaging over Slack (probably via the appropriate SDK repository channel)
 
+## Etiquette
+
+### Conversations
+
+- Conversations should be closed by the person who started them
```

Comment:
I think (here and below) "resolve" is the verb that GitHub uses, not "close".

Reply:
You're absolutely right. Thanks for pick up on that - I was trying to use the write nomenclature, but failed to do that here. I will fix that up...

Reply:
https://github.com/ably/engineering/pull/16/commits/816d709e3f20ce3cdf52f835be3756c0b7b56c67